CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -29,6 +33,15 @@ hopperbinGUI.Adornee = hopperbinPart
 hopperbinGUI.ToolPunchThroughDistance = 1000
 hopperbinGUI.Name = "HopperBinGUI"
 
+local healthbarBack = Instance.new("Frame", hopperbinGUI)
```

Comment:
This needs to show or hide based on the OnCoreGuiChanged for Healthbar coregui.
Also if backpack is not turned on but the Healthbar is turned on, does this module get required?